.vscode/launch.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
I presume that this wasn't intended to be in the pull request? :)
I can merge this and then delete it, or you can adjust the request, whichever is easier.